Jenkinsfile

```diff
@@ -1,69 +1,98 @@
+        }
     }
     stage('Copy UI into Server') {
-        copyArtifacts filter: 'quipucords-ui-dist.tar.gz', fingerprintArtifacts: true, projectName: "qpc_${VERSION}_ui_distribution", selector: lastCompleted()
-        sh "tar -xvf quipucords-ui-dist.tar.gz"
-        sh "cp -rf dist/client quipucords/"
-    	sh "cp -rf dist/templates quipucords/quipucords/"
-        sh "rm -rf dist"
+        steps {
+            copyArtifacts filter: 'quipucords-ui.*.tar.gz', fingerprintArtifacts: true, projectName: "quipucords-ui-build-job", selector: lastCompleted()
+            sh "tar -xvf quipucords-ui.*.tar.gz"
+            sh "ls -lah"
+            sh "cp -rf dist/client quipucords/"
+        	sh "cp -rf dist/templates quipucords/quipucords/"
+            sh "rm -rf dist"
+        }
     }
     stage('Build Docker Image') {
-        sh "ls -lta"
-        sh "cat Dockerfile"
+        steps {
+            sh "ls -lta"
+            sh "ls -lta quipucords"
+            sh "cat Dockerfile"
 
-        sh "git rev-parse HEAD > GIT_COMMIT"
-        sh 'cat GIT_COMMIT'
-        def commitHash = readFile('GIT_COMMIT').trim()
+            sh "git rev-parse HEAD > GIT_COMMIT"
+            sh 'cat GIT_COMMIT'
 
-        sh "sed -i s/BUILD_VERSION_PLACEHOLDER/${BUILD_VERSION}/g ${release_info_file}"
-        sh "sed -i s/BUILD_VERSION_PLACEHOLDER/${BUILD_VERSION}/g ${release_py_full_path}"
+            sh "sudo docker -D build --build-arg BUILD_COMMIT=`cat GIT_COMMIT` . -t $image_name"
+            sh "sudo docker save -o $tarfile $image_name"
+            sh "sudo chmod 755 $tarfile"
+            sh "sudo gzip -f --best $tarfile"
+            sh "sudo chmod 755 $targzfile"
 
-        sh "sudo docker -D build --build-arg BUILD_COMMIT=$commitHash . -t $image_name"
-        sh "sudo docker save -o $tarfile $image_name"
-        sh "sudo chmod 755 $tarfile"
-        sh "sudo gzip -f --best $tarfile"
-        sh "sudo chmod 755 $targzfile"
+            sh "sudo docker pull $postgres_image_name"
+            sh "sudo docker save -o $postgres_tarfile $postgres_image_name"
+            sh "sudo chmod 755 $postgres_tarfile"
 
-        sh "sudo docker pull $postgres_image_name"
-        sh "sudo docker save -o $postgres_tarfile $postgres_image_name"
-        sh "sudo chmod 755 $postgres_tarfile"
+            sh "mkdir $postgres_dir"
+            sh "mv $postgres_tarfile $postgres_dir"
+            sh "cp $postgres_license $rename_license"
+            sh "tar -zcvf $postgres_targzfile $postgres_dir"
+            sh "sudo chmod 775 $postgres_targzfile"
 
-        sh "mkdir $postgres_dir"
-        sh "mv $postgres_tarfile $postgres_dir"
-        sh "cp $postgres_license $rename_license"
-        sh "tar -zcvf $postgres_targzfile $postgres_dir"
-        sh "sudo chmod 775 $postgres_targzfile"
-
-        archiveArtifacts release_info_file
-        archiveArtifacts postgres_targzfile
-        archiveArtifacts targzfile
+            archiveArtifacts postgres_targzfile
```

Comment:
No need to archive the `release_info_file` anymore?

Reply:
We definitely need `release_info_file`.  The installation script will curl this.  All the logic associated with this must be preserved.  There are sed commands as well.